linera-version/tests/up_to_date.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use linera_version::VersionInfo;
+
+#[test]
+fn up_to_date() {
+    assert_eq!(
+        VersionInfo::get().unwrap().api_hashes(),
+        VersionInfo::default().api_hashes(),
+        "`linera-version` API hash cache out of date.\n\
+         Please update `linera-version/api-hashes.json` by running:\n\
+         $ cargo run linera-version > linera-version/api-hashes.json");
```

Comment:
nit: Maybe remove the `$` ? :)

Reply:
I think it's a pretty standard convention to clearly mark the following as a shell command to be executed, rather than commentary or command output. The command is wrong though :sweat_smile: